capstone/src/main/webapp/script.js

```diff
@@ -0,0 +1,42 @@
+google.charts.load('current', {packages: ['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function searchFunction() {
+  var input, filter, ul, li, a, i, txtValue;
+  input = document.getElementById('searchQuery');
+  filter = input.value.toUpperCase();
+  ul = document.getElementById("myUL");
+  li = ul.getElementsByTagName('li');
+
+  // Loop through all list items, and hide those who don't match the search query
+  for (i = 0; i < li.length; i++) {
+    a = li[i].getElementsByTagName("a")[0];
+    txtValue = a.textContent || a.innerText;
+    if (txtValue.toUpperCase().indexOf(filter) > -1) {
+      li[i].style.display = "";
+    } else {
+      li[i].style.display = "none";
+    }
+  }
+}
+
+function drawChart(){
+  var data = google.visualization.arrayToDataTable([
+    ['Genre', 'Fantasy & Sci Fi', 'Romance', 'Mystery/Crime', 'General',
```

Comment:
Leftover from copy pasting code? would be better if we cut out fluff, easier to understand whats going on that way